### DISCUSSION TITLE
QMediaPlayer undefined (reading "isActive')

### DISCUSSION THREAD

**Discussion by nyh1234444** [quasarframework/quasar-ui-qmediaplayer#362] [Q&A]
Good Days Everyone,

May i request for help regarding the Q media player. I currenty using Vue 3 and using this resource (https://qmediaplayer.netlify.app/developing/using-qmediaplayer) to do my project. But when i unabled to view the video with this error. Anyone can help me or gives my some idea. 

Thanks👍👍
![DFCE4771-D869-4859-BA8D-4857EB998C03](https://user-images.githubusercontent.com/113470191/189988588-2d618e25-d909-4052-8cae-9466244a73b8.jpeg)

**Comment by nucle** [comment]
Hi,
please can u post more information or some code snippet.

**Comment by genius192x** [comment]
> Good Days Everyone,
> 
> May i request for help regarding the Q media player. I currenty using Vue 3 and using this resource (https://qmediaplayer.netlify.app/developing/using-qmediaplayer) to do my project. But when i unabled to view the video with this error. Anyone can help me or gives my some idea.
> 
> Thanks👍👍 ![DFCE4771-D869-4859-BA8D-4857EB998C03](https://user-images.githubusercontent.com/113470191/189988588-2d618e25-d909-4052-8cae-9466244a73b8.jpeg)

Hi. Today i find the same problem with my own project. Solution: https://quasar.dev/quasar-plugins/app-fullscreen . In my case i just added this plugin in my quasar-user-options.js file (vue/vite project) like this
import {AppFullscreen} from 'quasar'
 export default {
  config: {
		brand: {
			primary: '#0F2557'
		}
	},
  plugins: {
    AppFullscreen
  }
}
